types/leaflet.locatecontrol/index.d.ts

```diff
@@ -3,9 +3,9 @@
 // Definitions by: Denis Carriere <https://github.com/DenisCarriere>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-/// <reference types="leaflet" />
+import L = require('leaflet');
 
-declare namespace L {
```

Comment:
This breaks the tests, the Type definition should be defined in the `namespace L`.

![image](https://user-images.githubusercontent.com/550895/27518230-165abd36-59a9-11e7-85fd-699b912a36e2.png)